### DISCUSSION TITLE
Add authorization via Telegram

### DISCUSSION THREAD

**Discussion by CodeExplorerX** [forem/forem#19267] [Feature Requests]
Adding Telegram authentication to Florem can have several advantages, and the following arguments can serve as a basis for justifying the need for such a solution:

1. Convenience for users. Authentication through Telegram can be a convenient way for users who already have a Telegram account to log into the site. This will allow them to quickly and securely access the site using their existing account.
2. Security. Telegram provides many features to protect user accounts, including two-factor authentication, which can be used as an additional level of security for accounts on Florem. This will help protect users from unauthorized access to their account and increase the level of security for Florem users.
3. Simple setup. Adding Telegram authentication to Florem can be a simple setup process that can be done using the Telegram API. This can reduce the burden on Florem developers and shorten the time spent developing authentication functionality.
4. Telegram's popularity. Telegram is one of the most popular messengers in the world with a large number of users. Adding Telegram authentication to Florem can attract new users interested in using this login method.

Overall, adding Telegram authentication to Florem can be a beneficial step in improving user experience and security for users.

**Comment by AlexaHills** [comment]
I must say that you are a genious and telegram is very useful and fast responding application. Could you suggest me some more features so that i can add my telegram to my project website based on decoration and How can i add telegram and link it with the website and project like [If](https://decortrendy.com/) you help me in telegram management for web it will be indeed very helpful thanks.

**Reply by CodeExplorerX** [reply]
Thank you for your kind words! I'm glad you found Telegram useful. Here are a few additional features:

1.  Telegram bots: You can create a bot in Telegram that will provide your website users with information about your project, answer user questions, and perform other functions.

3.  Sending notifications: With the Telegram API, you can send notifications to your website users, such as about changes in the project or new features that have been added.

4. Sending messages and files: You can allow users of your website to send messages and files through Telegram, which can be useful for exchanging information and feedback.

To add Telegram to your site, you first need to create a Telegram bot and get an API token. You can then use this token to integrate Telegram with your site. There are different ways to integrate Telegram with a website, and it depends on the technologies used on your site. Some platforms, such as WordPress, have plugins to integrate with Telegram. I can't help you here, I recommend finding a more experienced developer
